src/index.ts

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
```

Comment:
shebang ts-node but who gaf

Reply:
Wdym? This just runs in node when deployed as a bin